src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -1408,13 +1408,13 @@ void RNN::prepareParams() {
     primArgs[DNNL_ARG_SCRATCHPAD] = scratchpadMem->getPrimitive();
 }
 
-std::shared_ptr<MemoryDesc> RNN::getSrcMemDesc(const dnnl::primitive_desc& prim_desc, size_t idx) const {
-    (void)prim_desc;
+std::shared_ptr<MemoryDesc> RNN::getSrcMemDesc([[maybe_unused]] const dnnl::primitive_desc& prim_desc,
```

Comment:
```suggestion
std::shared_ptr<MemoryDesc> RNN::getSrcMemDesc(const dnnl::primitive_desc&,
```
In case of not used parameters make in unnamed, I think will be cleaner than adding attribiute

Reply:
There is a check [(readability-named-parameter)](https://clang.llvm.org/extra/clang-tidy/checks/readability/named-parameter.html) planned to be enabled that is going to get rid of unnamed parameters at all, that's why we have considered to use `[[maybe_unused]]` at this point.